R/render.R

```diff
@@ -45,6 +45,8 @@ render_page_html <- function(pkg, name, data = list(), depth = 0L) {
   data$opengraph <- utils::modifyList(data_open_graph(pkg), data$opengraph %||% list())
   data$footer <- data_footer(pkg)
 
+  pkg$development$version_tooltip <- tr_(pkg$development$version_tooltip)
```

Comment:
I don't think this will work correctly? `tr_()` always needs to be called with a literal string.

Reply:
Would change this. Would you agree if I remove the `tr_` on https://github.com/r-lib/pkgdown/blob/4c1668a04209e82db5367a420c894ff946379ca6/R/development.R ? It is not working and it may lead to confusion IMO.

Reply:
I'd recommend just hoisting that that code into this function. You can't do `tr_(foo)` because there's no way for `potools::po_extract()` to know what string `foo` contains.

Reply:
Now it is done